presto-docs/src/main/sphinx/release/release-0.210.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.210
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when aliasing columns of tables containing hidden columns (:issue:`11385`).
+* Fix correctness issue when ``GROUP BY DISTINCT`` terms contain references to the same column
+  using different syntactic forms (:issue:`11120`).
+* Fix failures when querying information_schema tables using capitalized names.
+* Remove min/max intersection heuristics from equality stats calculator.
+* Improve performance when converting between ``ROW`` types.
+* Remove user cpu time tracking.
```

Comment:
Capitalize CPU. What user visible effects does this have? SPI event changes? Config changes or removal?

Reply:
cc @arhimondr

Reply:
> Config changes or removal?

No config change at this step.

Reply:
> What user visible effects does this have?

This statistic is removed from the split statistics on the UI as well.

![selection_014](https://user-images.githubusercontent.com/5570988/45116337-649ed200-b120-11e8-8e08-859d05d2ae4a.png)